src/ui/component/fileViewer/view.jsx

```diff
@@ -54,15 +58,24 @@ export default function FileViewer(props: Props) {
       ? __("It looks like you deleted or moved this file. We're rebuilding it now. It will only take a few seconds.")
       : __('Loading');
 
-  function handleDrag(e, ui) {
-    const { x, y } = position;
-    const newX = x + ui.deltaX;
-    const newY = y + ui.deltaY;
-    setPosition({
-      x: newX,
-      y: newY,
-    });
-  }
+  const previousUri = usePrevious(uri);
+  const previousIsReadyToPlay = usePrevious(isReadyToPlay);
+  const isNewView = uri && previousUri !== uri && isPlaying;
+  const wasntReadyButNowItIs = isReadyToPlay && !previousIsReadyToPlay;
+
+  useEffect(() => {
+    if (isNewView) {
+      setPlayTime(Date.now());
+    }
```

Comment:
@tzarebczan brought up that we might not want to do play time (or keep play time separate) for non-streamable content.

Reply:
I'm not particularly concerned about this, I think it can be cleaned up in internal analytics.